hack/boilerplate/boilerplate.py

```diff
@@ -24,6 +24,7 @@
 import os
 import re
 import sys
+import datetime
```

Comment:
It looks like these imports are in alphabetical order.

Reply:
Done.